Error when starting catalog server in version 0.5.10

```
[main] ERROR org.radarcns.schema.service.SourceCatalogueServer - Cannot start server 
javax.servlet.ServletException: org.glassfish.jersey.servlet.ServletContainer-17a1e4ca@c0f22b31==org.glassfish.jersey.servlet.ServletContainer,jsp=null,order=-1,inst=true,async=true
	at org.eclipse.jetty.servlet.ServletHolder.initServlet(ServletHolder.java:620)
	at org.eclipse.jetty.servlet.ServletHolder.initialize(ServletHolder.java:413)
	at org.eclipse.jetty.servlet.ServletHandler.lambda$initialize$0(ServletHandler.java:749)
	at java.base/java.util.stream.SortedOps$SizedRefSortingSink.end(SortedOps.java:357)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:485)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.StreamSpliterators$WrappingSpliterator.forEachRemaining(StreamSpliterators.java:312)
	at java.base/java.util.stream.Streams$ConcatSpliterator.forEachRemaining(Streams.java:735)
	at java.base/java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:658)
	at org.eclipse.jetty.servlet.ServletHandler.initialize(ServletHandler.java:773)
	at org.eclipse.jetty.servlet.ServletContextHandler.startContext(ServletContextHandler.java:360)
	at org.eclipse.jetty.server.handler.ContextHandler.doStart(ContextHandler.java:844)
	at org.eclipse.jetty.servlet.ServletContextHandler.doStart(ServletContextHandler.java:275)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
	at org.eclipse.jetty.server.Server.start(Server.java:408)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:110)
	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:97)
	at org.eclipse.jetty.server.Server.doStart(Server.java:372)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
	at org.radarcns.schema.service.SourceCatalogueServer.start(SourceCatalogueServer.java:48)
	at org.radarcns.schema.service.SourceCatalogueServer$SourceCatalogueServiceCommand.execute(SourceCatalogueServer.java:66)
	at org.radarcns.schema.CommandLineApp.main(CommandLineApp.java:167)
	Suppressed: java.lang.IllegalStateException: !STOPPED
		at org.eclipse.jetty.server.handler.HandlerWrapper.destroy(HandlerWrapper.java:140)
		at org.radarcns.schema.service.SourceCatalogueServer.close(SourceCatalogueServer.java:38)
		at org.radarcns.schema.service.SourceCatalogueServer$SourceCatalogueServiceCommand.execute(SourceCatalogueServer.java:65)
		... 1 more

Caused by: A MultiException has 2 exceptions.  They are:
1. com.fasterxml.jackson.databind.JsonMappingException: Scala module 2.10.2 requires Jackson Databind version >= 2.10.0 and < 2.11.0
2. java.lang.IllegalStateException: Unable to perform operation: create on org.glassfish.jersey.jackson.internal.DefaultJacksonJaxbJsonProvider

	at org.jvnet.hk2.internal.ClazzCreator.create(ClazzCreator.java:368)
	at org.jvnet.hk2.internal.SystemDescriptor.create(SystemDescriptor.java:463)
	at org.jvnet.hk2.internal.SingletonContext$1.compute(SingletonContext.java:59)
	at org.jvnet.hk2.internal.SingletonContext$1.compute(SingletonContext.java:47)
	at org.glassfish.hk2.utilities.cache.Cache$OriginThreadAwareFuture$1.call(Cache.java:74)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at org.glassfish.hk2.utilities.cache.Cache$OriginThreadAwareFuture.run(Cache.java:131)
	at org.glassfish.hk2.utilities.cache.Cache.compute(Cache.java:176)
	at org.jvnet.hk2.internal.SingletonContext.findOrCreate(SingletonContext.java:98)
	at org.jvnet.hk2.internal.Utilities.createService(Utilities.java:2102)
	at org.jvnet.hk2.internal.ServiceHandleImpl.getService(ServiceHandleImpl.java:93)
	at org.jvnet.hk2.internal.ServiceHandleImpl.getService(ServiceHandleImpl.java:67)
	at org.glassfish.jersey.inject.hk2.AbstractHk2InjectionManager.lambda$getAllServiceHolders$0(AbstractHk2InjectionManager.java:136)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.LinkedList$LLSpliterator.forEachRemaining(LinkedList.java:1239)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578)
	at org.glassfish.jersey.inject.hk2.AbstractHk2InjectionManager.getAllServiceHolders(AbstractHk2InjectionManager.java:140)
	at org.glassfish.jersey.inject.hk2.ImmediateHk2InjectionManager.getAllServiceHolders(ImmediateHk2InjectionManager.java:30)
	at org.glassfish.jersey.internal.inject.Providers.getServiceHolders(Providers.java:307)
	at org.glassfish.jersey.internal.inject.Providers.getCustomProviders(Providers.java:151)
	at org.glassfish.jersey.message.internal.MessageBodyFactory.initialize(MessageBodyFactory.java:219)
	at org.glassfish.jersey.message.internal.MessageBodyFactory$MessageBodyWorkersConfigurator.postInit(MessageBodyFactory.java:114)
	at org.glassfish.jersey.server.ApplicationHandler.lambda$initialize$2(ApplicationHandler.java:349)
	at java.base/java.util.Arrays$ArrayList.forEach(Arrays.java:4390)
	at org.glassfish.jersey.server.ApplicationHandler.initialize(ApplicationHandler.java:349)
	at org.glassfish.jersey.server.ApplicationHandler.lambda$initialize$1(ApplicationHandler.java:293)
	at org.glassfish.jersey.internal.Errors.process(Errors.java:292)
	at org.glassfish.jersey.internal.Errors.process(Errors.java:274)
	at org.glassfish.jersey.internal.Errors.processWithException(Errors.java:232)
	at org.glassfish.jersey.server.ApplicationHandler.initialize(ApplicationHandler.java:292)
	at org.glassfish.jersey.server.ApplicationHandler.<init>(ApplicationHandler.java:259)
	at org.glassfish.jersey.servlet.WebComponent.<init>(WebComponent.java:311)
	at org.glassfish.jersey.servlet.ServletContainer.init(ServletContainer.java:154)
	at org.glassfish.jersey.servlet.ServletContainer.init(ServletContainer.java:347)
	at javax.servlet.GenericServlet.init(GenericServlet.java:244)
	at org.eclipse.jetty.servlet.ServletHolder.initServlet(ServletHolder.java:602)
	... 22 more
Caused by: com.fasterxml.jackson.databind.JsonMappingException: Scala module 2.10.2 requires Jackson Databind version >= 2.10.0 and < 2.11.0
	at com.fasterxml.jackson.module.scala.JacksonModule.setupModule(JacksonModule.scala:61)
	at com.fasterxml.jackson.module.scala.JacksonModule.setupModule$(JacksonModule.scala:46)
	at com.fasterxml.jackson.module.scala.DefaultScalaModule.setupModule(DefaultScalaModule.scala:17)
	at com.fasterxml.jackson.databind.ObjectMapper.registerModule(ObjectMapper.java:818)
	at com.fasterxml.jackson.databind.ObjectMapper.registerModules(ObjectMapper.java:1020)
	at com.fasterxml.jackson.databind.ObjectMapper.findAndRegisterModules(ObjectMapper.java:1098)
	at org.glassfish.jersey.jackson.internal.DefaultJacksonJaxbJsonProvider.findAndRegisterModules(DefaultJacksonJaxbJsonProvider.java:44)
	at org.glassfish.jersey.jackson.internal.DefaultJacksonJaxbJsonProvider.<init>(DefaultJacksonJaxbJsonProvider.java:33)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at org.glassfish.hk2.utilities.reflection.ReflectionHelper.makeMe(ReflectionHelper.java:1356)
	at org.jvnet.hk2.internal.ClazzCreator.createMe(ClazzCreator.java:248)
	at org.jvnet.hk2.internal.ClazzCreator.create(ClazzCreator.java:342)
	... 61 more
```

--- a/java-sdk/build.gradle
+++ b/java-sdk/build.gradle
@@ -17,7 +17,7 @@ subprojects {
     apply plugin: 'idea'
 
     // Configuration
-    version = '0.5.10'
+    version = '0.5.10.1'
     group = 'org.radarcns'
     ext.githubRepoName = 'RADAR-base/RADAR-Schemas'
 

--- a/java-sdk/radar-schemas-tools/build.gradle
+++ b/java-sdk/radar-schemas-tools/build.gradle
@@ -31,8 +31,8 @@ ext {
     description = 'RADAR Schemas specification and validation tools.'
 
     argparseVersion = '0.8.1'
-    jacksonVersion = '2.11.0'
-    jacksonYamlVersion = '2.11.0'
+    jacksonVersion = '2.10.4'
+    jacksonYamlVersion = '2.10.4'
     jerseyVersion = '2.31'
     jettyVersion = '9.4.29.v20200521'
     junitVersion = '4.13'
@@ -53,13 +53,21 @@ repositories {
 }
 
 dependencies {
-    api group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: jacksonVersion
     api group: 'javax.validation', name: 'validation-api', version: javaxValidationVersion
     api group: 'org.apache.kafka', name: 'kafka_2.12', version: kafkaVersion
     // temporary version override to avoid netty vulnerability. Can be removed as soon as Kafka has an updated dependency
     implementation group: 'org.apache.zookeeper', name: 'zookeeper', version: '3.5.8'
 
-    implementation group: 'com.fasterxml.jackson.dataformat', name: 'jackson-dataformat-yaml', version: jacksonYamlVersion
+    api ("com.fasterxml.jackson.core:jackson-databind") {
+      version {
+        strictly jacksonVersion
+      }
+    }
+    implementation ("com.fasterxml.jackson.dataformat:jackson-dataformat-yaml") {
+      version {
+        strictly jacksonYamlVersion
+      }
+    }
     implementation project(':radar-schemas-commons')
     implementation group: 'org.radarbase', name: 'radar-commons', version: radarCommonsVersion
     implementation group: 'org.radarbase', name: 'radar-commons-server', version: radarCommonsVersion

